extensions/wikia/CommunityPage/styles/components/_help.scss

```diff
@@ -2,57 +2,67 @@
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/base/colors';
 @import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
 @import 'skins/shared/mixins/flexbox';
+@import 'variables';
 
 $small-icon-size: 18px;
-$big-icon-size: 48px;
-$help-module-margin: 12px;
-$help-module-padding-bottom: 20px;
-$policy-link-padding-top: 18px;
-$policy-link-margin-top: 14px;
-
-$help-module-link-line-height: 2;
+$big-icon-size: 30px;
+$help-module-title-padding: 3px;
+$help-module-link-line-height: 18px;
```

Comment:
Isn't it a default design-system line height? I think it is defined as `$line-height` variable.